File: CoreJava/Shoppingcart-app/src/com/techlabs/shoppingcart/Customer.java
package com.techlabs.shoppingcart;

import java.util.*;

public class Customer {

	private int id;
	private String name;

	private List<Order> orders = new ArrayList<Order>();

	public Customer(int id, String name) {
		super();
		this.id = id;
		this.name = name;
	}

	public int getId() {
		return id;
	}

	public String getName() {
		return name;
	}

	public List<Order> getOrders() {
		return orders;
	}

	public void addOrder(Order order) {
		orders.add(order);
	}

}

File: DotNET/C#/ShoppingCartApp/ShoppingCartApp/Customer.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace ShoppingCartApp
{
    class Customer
    {
        private int _id;
        private String _name;
        private List<Order> _orders = new List<Order>();

        public Customer(int id, String name)
        {
            _id = id;
            _name = name;
        }

        public int Id
        {
            get
            {
                return _id;
            }
        }

        public String Name
        {
            get
            {
                return _name;
            }
        }

        public List<Order> Order
        {
            get
            {
                return _orders;
            }

        }

        public void AddOrder(Order order)
        {
            _orders.Add(order);
        }
    }
}
